src/settings/MappingProfiles/MappingProfiles.js

```diff
@@ -1,46 +1,47 @@
-import React, {
-  Component,
-  Fragment,
-} from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import queryString from 'query-string';
 import {
   get,
-  noop,
+  omit,
 } from 'lodash';
 
-import {
-  IntlConsumer,
-  stripesConnect,
-} from '@folio/stripes/core';
+import { stripesConnect } from '@folio/stripes/core';
 import { makeQueryFunction } from '@folio/stripes/smart-components';
 import { Checkbox } from '@folio/stripes/components';
 
 import {
-  listTemplate,
-  SearchAndSort,
-} from '../../components';
-import { ViewMappingProfile } from './ViewMappingProfile';
-import { SettingPage } from '../SettingPage';
-import {
-  trimSearchTerm,
   withCheckboxList,
   checkboxListShape,
 } from '../../utils';
 import { ENTITY_KEYS } from '../../utils/constants';
+import { ListView } from '../../components';
+import { ViewMappingProfile } from './ViewMappingProfile';
+// import { MappingProfilesForm } from 'MappingProfilesForm';
```

Comment:
Is it needed?

Reply:
Yes it is in the next ticket which Alexander will perform.

Reply:
I propose to remove then that stuff and wait for the next story.